app/src/main/java/com/cortado/minionify/mode/AppMode.java

```diff
@@ -0,0 +1,28 @@
+package com.cortado.minionify.mode;
+
+public enum AppMode {
+    MINION,
+    MANAGER;
+
+    public static int toInteger(AppMode appMode) {
```

Comment:
instead of the switch, use a field `mValue` (search for enums with fields on google if you dont know it)

Reply:
Done. Didn't know that exists in Java!